src/components/organelle-wrapper.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+declare var Organelle: any;
+
+interface OrganelleWrapper {
+  name: string,
+  modelProperties: any,
+  showBinding: boolean,
+  doAddHormone: boolean,
+  addEnzyme: boolean,
+  setActiveAssay: Function,
+  currentView: any,
+  setGraphState: Function
+}
+
+class OrganelleWrapper extends React.Component<any, any> {
+    model: any;
+  constructor(props: OrganelleWrapper) {
+    super(props);
+    this.model = null;
+    this.addHormone = this.addHormone.bind(this);
+  }
+
+  componentDidMount() {
+    const {modelProperties} = this.props;
+
+    Organelle.createModel({
+      element: this.props.name,
+      background: {
+        file: "assets/melanocyte.svg",
+        selector: "#cell"
+      },
+      properties: modelProperties,
+      calculatedProperties: {},
+      clickHandlers: [
+        {
+          selector: "#melanocyte_x5F_cell",
+          action: this.organelleClick.bind(this, "cytoplasm")
+        },
+        {
+          selector: "#nucleus_x5F_A",
+          action: this.organelleClick.bind(this, "nucleus")
+        },
+      ],
+      species: [
+        "organelles/hexagon.yml",
+        "organelles/triangle.yml",
+        "organelles/g-protein.yml",
+        "organelles/g-protein-part.yml"
+      ]
+    }).then((m:any) => {
+      this.model = m;
+
+      this.model.setTimeout(() => {
+        let cell:any = document.querySelector(`#${this.props.name} #cellshape_0_Layer0_0_FILL`)
+        if (cell) {
+            cell.style["fill-opacity"] = 0.5;
+            cell.style["fill"] = "rgb(241,212,151)";
+        }
+      }, 10);
```

Comment:
I took this from the `mlm-demos` project - I think the idea of the timeout is to make sure the SVG is loaded before trying to manipulate it, but of course the timeout isn't always long enough so sometimes the code doesn't run properly. Should the `create-model` promise should only resolve when the SVG is loaded? Then we should be able to drop this timeout entirely, I think.